.github/workflows/tests.yml

```diff
@@ -13,6 +13,10 @@ jobs:
         with:
           toolchain: stable
       - uses: Swatinem/rust-cache@v2
+      - name: Install cargo insta
+        uses: taiki-e/install-action@v2
```

Comment:
Plucked from https://github.com/astral-sh/ruff/blob/fc8738f52af124ab54df61c7a158bf46fe8e7532/.github/workflows/ci.yaml#L121.

I haven't seen this yet https://github.com/taiki-e/install-action.